File: Java Day - 02/SimpleCalculator.java
import java.util.Scanner;

class Calculator {

    void add(double num1, double num2) {
        System.out.println("The sum of two number is : " + (num1 + num2));
    }

    void subtract(double num1, double num2) {
        System.out.println("The sub of two number is : " + (num1 - num2));
    }

    void multiply(double num1, double num2) {
        System.out.println("The multification of two number is : " + (num1 * num2));
    }

    void divide(double num1, double num2) {
        if (num2 != 0)
            System.out.println("The division of two number is : " + (num1 / num2));
        else
            System.out.println("Error! Division by zero.");
    }
}

public class SimpleCalculator {

    public static void main(String[] args) {
        Scanner myInput = new Scanner(System.in);

        System.out.println("Enter first number: ");
        double num1 = myInput.nextDouble();

        System.out.println("Enter second number: ");
        double num2 = myInput.nextDouble();

        
        Calculator obj = new Calculator();
        obj.add(num1, num2);
        obj.subtract(num1, num2);
        obj.multiply(num1, num2);
        obj.divide(num1, num2);
        myInput.close();
    }
}

File: Java Day - 02/210137/simpleCalculator.cpp
#include <bits/stdc++.h>
using namespace std;
class Calculator
{
public:
    int add(int a, int b)
    {
        return a + b;
    }
    int subtract(int a, int b)
    {
        return a - b;
    }
    int multiply(int a, int b)
    {
        return a * b;
    }
    int divide(int a, int b)
    {
        return a / b;
    }
};

int main()
{
    Calculator obj;
    int a, b;
    cout << "Enter two numbers: ";
    cin >> a >> b;

    cout << "Addition of two numbers : " << obj.add(a, b) << '\n';
    cout << "Subtraction of two numbers : " << obj.subtract(a, b) << '\n';
    cout << "Multiplication of two numbers : " << obj.multiply(a, b) << '\n';
    cout << "Division of two numbers : " << obj.divide(a, b) << '\n';
}
